### PR TITLE
ED-2970 Add append slash

### PR DESCRIPTION
**Pull request by sdonk**

### CODE REVIEW

File: ui/settings.py [hunk 1]
```diff
@@ -30,6 +30,7 @@
 # PaaS, we can open ALLOWED_HOSTS
 ALLOWED_HOSTS = ['*']
 
+APPEND_SLASH = True
```
Comment (richtier) [inline ui/settings.py]: append slash causes 302 to happen if the requested url does not end in slash.

this redirect will cause any data in a POST request to be lost.

in github.com/iuktrade/directory-api-client-external it seems all the urls end in slashes, so this should not be a problem.

Do all the forms still work in FAB?

File: ui/urls.py [hunk 2]
```diff
@@ -120,7 +120,7 @@
         name='verify-companies-house-callback'
     ),
     url(
-        r'^confirm-company-address$',
+        r'^confirm-company-address/$',
```
Comment (richtier) [inline ui/urls.py]: to be explicit can you add the slashes to all the urls please.

File: ui/settings.py (hunk 1 above)
Comment (konradko) [reply ui/settings.py]: Append slash only works when you request a URL without trailing slash > it is not found > one with slash is > 302. So all the endpoints that are not using slash should still work just fine.

Best scenario would be to use slash everywhere, just to be consistent. But it is a hard requirement only for the public facing URLs

File: ui/urls.py (hunk 2 above)
Comment (konradko) [reply ui/urls.py]: only public facing ones please (so no api endpoints)

**Pull request merged**